examples/vg-specs/area.vg.json

```diff
@@ -33,6 +33,9 @@
                     ],
                     "fields": [
                         "count"
+                    ],
+                    "as": [
```

Comment:
Why do we add this all of a sudden? Is it necessary? I think this should only be generated if we don't use the default.